lib/theme_check/node.rb

```diff
@@ -132,12 +162,81 @@ def position
       )
     end
 
+    def start_token
+      return "" if inside_liquid_tag?
+      output = ""
+      output += "{{" if variable?
+      output += "{%" if tag?
```

Comment:
Can't be a variable & tag at the same time:

```suggestion
      if variable?
        output += "{{"
      elsif tag?
        output += "{%"
      end
```

Same bellow.